packages/flutter_tools/lib/src/application_package.dart

```diff
@@ -330,16 +332,8 @@ class ApkManifestData {
         final Map<String, String> entries = <String, String>{};
         map[name] = entries;
 
-        for (String entry in line.split(' ')) {
-          entry = entry.trim();
-          if (entry.isNotEmpty && entry.contains('=')) {
-            final int split = entry.indexOf('=');
-            final String key = entry.substring(0, split);
-            String value = entry.substring(split + 1);
-            if (value.startsWith("'") && value.endsWith("'"))
-              value = value.substring(1, value.length - 1);
-            entries[key] = value;
-          }
+        for (Match m in keyValueRegex.allMatches(line)) {
+          entries[m.group(1)] = m.group(2);
```

Comment:
this used to support foo=bar and foo='bar', but not only supports foo='bar', is that intentional?

Reply:
also this handles `foo='bar baz=quux'` differently than what we did before, and will find a=b in `zz="xx a='b' yy"`

we should add tests for these cases to make sure they do what we want, in case someone "fixes" the code to be like it was before.

Reply:
(specifically single quotes - it will not use double quotes as in the `zz="xx a='b' yy"` case)